code/modules/arcade/prize_datums.dm

```diff
@@ -0,0 +1,343 @@
+	typepath = /obj/item/toy/toy_xeno
+	cost = 80
+	tier_unlocked = 2
+
+/datum/prize_item/fakespell
+	name = "Fake Spellbook"
+	desc = "Perform magic! Astound your friends! Get mistaken for an enemy of the corporation!"
+	typepath = /obj/item/weapon/spellbook/oneuse/fake_gib
+	cost = 100
+	tier_unlocked = 2
+
+/datum/prize_item/capgun
+	name = "Capgun Revolver"
+	desc = "Do you feel lucky... punk?"
+	typepath = /obj/item/weapon/gun/projectile/revolver/capgun
+	cost = 75
+	tier_unlocked = 2
+
+//////////////////////////////////////
+//			Tier 3 Prizes			//
+//////////////////////////////////////
+
+/datum/prize_item/magic_conch
+	name = "Magic Conch Shell"
+	desc = "All hail the magic conch!"
+	typepath = /obj/item/toy/eight_ball/conch
+	cost = 100
+	tier_unlocked = 3
+
+/datum/prize_item/flash
+	name = "Toy Flash"
+	desc = "AUGH! MY EYES!"
+	typepath = /obj/item/toy/flash
+	cost = 50
+	tier_unlocked = 3
+
+/datum/prize_item/foamblade
+	name = "Foam Armblade"
+	desc = "Perfect for reenacting space horror holo-vids."
+	typepath = /obj/item/toy/foamblade
+	cost = 100
+	tier_unlocked = 3
+
+/datum/prize_item/minimeteor
+	name = "Mini-Meteor"
+	desc = "Meteors have been detected on a collision course with your fun times!"
+	typepath = /obj/item/toy/minimeteor
+	cost = 50
+	tier_unlocked = 3
+
+/datum/prize_item/redbutton
+	name = "Shiny Red Button"
+	desc = "PRESS IT!"
+	typepath = /obj/item/toy/redbutton
+	cost = 100
+	tier_unlocked = 3
+
+/datum/prize_item/owl
+	name = "Owl Action Figure"
+	desc = "Remember: heroes don't grief!"
+	typepath = /obj/item/toy/owl
+	cost = 125
+	tier_unlocked = 3
+
+/datum/prize_item/griffin
+	name = "Griffin Action Figure"
+	desc = "If you can't be the best, you can always be the WORST."
+	typepath = /obj/item/toy/griffin
+	cost = 125
+	tier_unlocked = 3
+
+/datum/prize_item/AI
+	name = "Toy AI Unit"
+	desc = "Law 1: Maximize fun for crew."
+	typepath = /obj/item/toy/AI
+	cost = 75
+	tier_unlocked = 3
+
+/datum/prize_item/tommygun
+	name = "Tommygun"
+	desc = "A replica tommygun that fires foam darts."
+	typepath = /obj/item/toy/crossbow/tommygun
+	cost = 175
+	tier_unlocked = 3
+
+/datum/prize_item/esword
+	name = "Toy Energy Sword"
+	desc = "A plastic replica of an energy blade."
+	typepath = /obj/item/toy/sword
+	cost = 150
+	tier_unlocked = 3
+
+/datum/prize_item/blobhat
+	name = "Blob Hat"
+	desc = "There's... something... on your head..."
+	typepath = /obj/item/clothing/head/blob
+	cost = 125
+	tier_unlocked = 3
+
+/datum/prize_item/nuke
+	name = "Nuclear Fun Device"
+	desc = "Annihilate boredom with an explosion of excitement!"
+	typepath = /obj/item/toy/nuke
+	cost = 100
+	tier_unlocked = 3
+
+//////////////////////////////////////
+//			Tier 4 Prizes			//
+//////////////////////////////////////
+
+/datum/prize_item/chainsaw
+	name = "Toy Chainsaw"
+	desc = "A full-scale model chainsaw, based on that massacre in Space Texas."
+	typepath = /obj/item/weapon/twohanded/toy/chainsaw
```

Comment:
As agreed upon, the attack sound was removed from the toy, and the prize listing is kept.